docs/src/pages/[platform]/getting-started/migration/migration.angular.mdx

```diff
@@ -25,9 +25,8 @@ yarn add aws-amplify @aws-amplify/ui-angular@3.x
 
 The following breaking changes were introduced in `@aws-amplify/ui-angular` major version `3.0`:
 
-#### Automatic signin on signup logic moved to `aws-amplify`. 
```

Comment:
Why is this removed? The two steps below are related subitems of this. Is there a better way to say this?

Reply:
Oh yeah, that was my attempt to flatten this list. 

My thought was that "Automatic signin on signup logic moved to `aws-amplify`" is more of an internal change. We have always done `autoSignIn` from customer's POV. That's why I went ahead and removed it, but what do you think?